File: practica2/api/connection/conn.js
const mysql = require("mysql2");

const { HOSTDB, USERDB, PASSDB, NAMEDB, PORTDB} = process.env;

const connection = mysql.createConnection({
    host: HOSTDB,
    user: USERDB,
    password: PASSDB,
    database: NAMEDB,
    port: PORTDB
});

module.exports = connection;

File: practica2/backend/db/conn.go
package db

import (
	"database/sql"
	"log"
	"os"

	_ "github.com/go-sql-driver/mysql"
	"github.com/joho/godotenv"
)

func Conexion() (*sql.DB, error) {
	err := godotenv.Load()
	if err != nil {
		log.Fatal("Error: No se puede cargar el archivo .env")
	}

	return sql.Open("mysql", os.Getenv("USERDB")+":"+os.Getenv("PASSDB")+"@tcp("+os.Getenv("HOSTDB")+":"+os.Getenv("PORTDB")+")/"+os.Getenv("NAMEDB"))
	//return sql.Open("mysql", "root:root@tcp(localhost:3306)/sopes")
}
